ansible-ipi-install/roles/bootstrap/tasks/10_load_inv.yml

```diff
@@ -307,16 +306,6 @@
     msg: "Not a supported configuration for workers"
   when: "'r620' in worker_types and 'r630' in worker_types"
 
-- name: set fact for ocp nodes mgmt
```

Comment:
Are you removing these since we are not using them anywhere?

Reply:
yes